### PR TITLE
only display time range in statement PDF when range is available (#6896, #6897)

### PR DESCRIPTION
**Pull request by willy-b**
for #6896 
(replaces and comes from discussion in #6897 and in https://github.com/willy-b/browser-laptop/commit/a510c87d1598b6e75043bf2924e36643355e4e7e#commitcomment-20667505 , where @mrose17 reviewed this change)

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.

Test Plan:
- either have a payment history or generate one with `npm run add-simulated-payment-history`
- open Brave -> Preferences -> Payments (or navigate to `about:preferences#payments`)
- click "View Payment History"
- open your first payment receipt PDF
- observe there is no date range present
- open a later payment receipt PDF
- observe a time range is present

Example: Two consecutive statements
-  where 1st has no time range (since 1st payment):
statementpdf_norange

- 2nd does have a time range (begins with date of last statement):
statementpdf_yesrange

### CODE REVIEW

**Comment by NejcZdovc** [comment]
I am not sure that first commit is part of this PR

File: tools/utilAppRunner.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function runUtilApp (cmd, file, stdioOptions) {
   }
   cmd = cmd.split(' ')
   if (process.env.NODE_ENV === 'development') {
-    cmd.push('--user-data-dir=brave-development')
+    cmd.push('--user-data-dir=../../../brave-development')
```
Comment (mrose17) [inline tools/utilAppRunner.js]: @willy-b - what's the motivation behind this change, i think it might break some other tests...

Comment (willy-b) [inline tools/utilAppRunner.js]: Good question, @mrose17. This change ensures Brave has the right path to `ledger-state.json` when `NODE_ENV` is set to `development`.

Reason:
For these utility scripts, the userData path is retrieved via `app.getPath(\"userData\")` run in `./tools/lib/utilApp/index.js` (three directory levels removed from the base of the repo).
Before this change, `./tools/utilAppRunner.js` was running brave with `--user-data-dir=brave-development`, which would cause Brave to look in `./tools/lib/utilApp/brave-development` instead of `./brave-development`.

What tests do you think it might break?

**Comment by bsclifton** [comment]
@mrose17 have you had a chance to check this PR out? Once it's in a state that you approve of, I'll gladly review / check tests and if good to go, merge 😄

**Comment by mrose17** [comment]
it is fine for merge.

**Comment by bsclifton** [comment]
Most tests [passed CI](https://travis-ci.org/brave/browser-laptop/builds/196755191); the ones that didn't I re-ran.  Some (like punycode) are perma-failing... but not related to this change.  Merging 😄

**Comment by mrose17** [comment]
thanks!
